docs/tutorials/galga.md

```diff
@@ -1,22 +1,41 @@
 # Galga
 
+
 ### ~button /#tutorial:/tutorials/galga
 
 Try this tutorial!
 
 ### ~
 
+### @autoexpandOff true
+
 ## Introduction @unplugged
 
-Fly your space plane through the oncoming bogey spacecraft. Can you survive the continuous attack? You start with three lives but you gain more by firing on the enemy ships. You lose a life with every collision, so try to blast away the enemy before they hit you.
+**In this tutorial, you'll create a space plane 
+that can fly through bogey spacecraft.**
+
+Can you survive the continuous attack? 
+
+---
 
 ![Space plane and attacking spacecraft](/static/tutorials/galga.gif)
 
 ## Step 1
 
-Add code to create a ``||sprites:sprite||``, rename it as ``||variables:spacePlane||``, and draw a plane or some kind of flying object in it.
+✈️ First, we're going to need a **space plane** [__*sprite*__](#sprote "2-D image that moves on the screen") ✈️
```

Comment:
this line looks like it might be the problem (sprote -> sprite)

Reply:
I don't think so. These are internal tags meant to keep the links from going anywhere, they've been in here since the first redo...but if they *are* the problem, it means having to make a bookmark in every file that has definitions in it.  Messy!

Reply:
Now that I think about it, all these files have already passed the new tests recently with these tags in them, so it's probably something else.